docker-compose.yml

```diff
@@ -90,6 +92,40 @@ services:
       - 3000:3000
     restart: unless-stopped
 
+  lnd:
+    container_name: lnd
+    image: lightninglabs/lnd:v0.15.5-beta
+    depends_on:
+      - bitcoin
+    command:
+      - --configfile=/var/lib/lnd/lnd.conf
+    volumes:
+      - ./faucet/lnd.conf:/var/lib/lnd/lnd.conf
+      - ./faucet/macaroons.db:/root/.lnd/data/chain/bitcoin/regtest/macaroons.db
```

Comment:
This will create a directory locally. Is this needed or can we use a virtual folder?

Reply:
That we mount the `macaroons.db` like that is not ideal, but I did not find a better way to ensure that the `admin.macaroon` will not change on every fresh start.